package.json

```diff
@@ -94,20 +95,20 @@
     "jest-cli": "^26.6.3",
     "lint-staged": "^10.5.3",
     "prettier": "^2.2.1",
-    "relay-compiler": "^10.0.1",
-    "relay-runtime": "^10.0.1",
-    "relay-test-utils-internal": "^10.0.1",
+    "relay-compiler": "^11.0.0",
+    "relay-runtime": "^11.0.0",
+    "relay-test-utils-internal": "^11.0.0",
     "ts-jest": "^26.4.4",
     "ts-node": "^9.1.1",
     "tslint": "^6.1.3",
     "tslint-config-prettier": "^1.18.0",
     "typescript": "4.1.5"
   },
   "peerDependencies": {
-    "@types/react-relay": ">=7.0.0",
-    "@types/relay-runtime": ">=6.0.7",
-    "relay-compiler": ">=9.0.0",
-    "relay-runtime": ">=9.0.0",
+    "@types/react-relay": "^11.0.0",
+    "@types/relay-runtime": ">=10.1.10",
+    "relay-compiler": ">=11.0.0",
+    "relay-runtime": ">=11.0.0",
```

Comment:
Based on this I'm assuming this change will likely need to be apart of a major version bump.

Reply:
we can ship this and the remaining breaking changes I have in my branch as a new major version next month while also dropping node 10

Reply:
That's true, yeah. Given that there's an extra entry point (a sort of opt-in if you will) I feel like we could likely loosen this requirement with the caveat of course that if you're using hooks version you'll need to be on v11. Not sure if it's worth encoding that as a dependency or runtime constraint vs just documenting it.